packages/0xcert-ethereum-generic-provider/src/core/mutation.ts

```diff
@@ -239,6 +239,77 @@ export class Mutation extends EventEmitter implements MutationBase {
     return this.resolveCurrentState();
   }
 
+  /**
+   * Retries mutation with higher gas price if possible. It uses providers retryGasMultiplier to
+   * calculate gas price.
+   * @notice Return error if mutation is already accepted into the Blockchain.
+   */
+  public async retry() {
+    const tx = await this.getTransactionObject();
+    if (!tx) {
+      throw new Error('Mutation not found');
+    } else if (tx.blockNumber) {
+      throw new Error('Mutation already acceped into the Blockchain');
+    }
+
+    const gasPrice = await this._provider.post({
+      method: 'eth_gasPrice',
+      params: [],
+    });
+    const oldGasPrice = tx.gasPrice;
+    const retryGasPrice = gasPrice.result * this._provider.retryGasPriceMultiplier;
+    // We first calculate new gas price based on current network conditions and
+    // retryGasPriceMultiplier if calculated gas price is lower then the original gas price then we
```

Comment:
// retryGasPriceMultiplier if calculated gas price is lower than the original gas price, then we